Library/Formula/qt.rb

```diff
@@ -58,6 +59,12 @@ def install
       end
     end
 
+    if MacOS.version >= :el_capitan
+      args << "-openssl-linked"
+      args << "-I" << "/usr/local/opt/openssl/include"
```

Comment:
`Formula["openssl"].opt_include` -- we don't know that we're installed to /usr/local.